azure/init.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# fail if trying to reference a variable that is not set.
+set -u
+# exit immediately if a command fails
+set -e
+# fail in a pipeline if any of the commands fails
+set -o pipefail
+
+# in redhat we need to enable default port for postgres
+firewall-cmd --add-port=5432/tcp
+
+# install pip since we will use it to install dependencies
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+
+# install git to clone the repository
+yum install -y git
+
+# this is the username in our instances
+TARGET_USER=pguser
+
+# attach disk and mount it for data
+mkfs.ext4 -F /dev/sdc
```

Comment:
How are we sure that it'd be also be `/dev/sdc`? Also, don't we need to add it do `/etc/fstab` to keep it resistant to restarts? Some pointers for [that](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-using-volumes.html)

Reply:
```
Normally, when we create a new Azure VM, the VM's OS disk name is /dev/sda, the temporary disk name is /dev/sdb.

Then we add a new data disk to this VM, by default, the new data disk name will be /dev/sdc.
```

So in our case it will be always `/dev/sdc`, If we want to attach more than one disk we might need to change this, so I think it is better for now to create an issue covering that(So if someone wants to add multiple disks, they know what to change)

Reply:
#121

Reply:
Well, the note is just from a random user and there is even an answer below with `The disk usually becomes /dev/sdc but I've seen it as /dev/sda`.

I'd be in favor of spending some more time and finding a programatic solution instead of hardcoding it. Some pointers are [here](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/attach-disk-portal)

On ec2, I've seen that the disk is attached to other `/dev/sdX`s. And, I don't see any guarantees that `sdc` always the right place.

Reply:
I have added the logic to find it programatically.